HW_one/Task_2.py

```diff
@@ -0,0 +1,12 @@
+# Задача 4: Петя, Катя и Сережа делают из бумаги журавликов. Вместе
+# они сделали S журавликов. Сколько журавликов сделал каждый
+# ребенок, если известно, что Петя и Сережа сделали одинаковое
+# количество журавликов, а Катя сделала в два раза больше журавликов,
+# чем Петя и Сережа вместе?
+
+print ('Введите количество журавликов')
+s = int(input())
+s /= 6
```

Comment:
Нужно использовать целочисленное деления, иначе у вас получится дробное число, а такого быть не может: журавлики всегда целые.
```suggestion
s //= 6
```